src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    @Override
+    public String toString() {
+        return "FruitTransaction{"
+                + "operation=" + operation
+                + ", fruit='" + fruit + '\''
+                + ", quantity=" + quantity
+                + '}';
+    }
+
+    public enum Operation {
+        BALANCE("b"),
+        SUPPLY("s"),
+        PURCHASE("p"),
+        RETURN("r");
+
+        private String code;
+
+        Operation(String code) {
+            this.code = code;
+        }
+
+        public String getCode() {
+            return code;
+        }
+
```

Comment:
you can add one more method for your convenience when you need to use parser.

public static Operation getByCode(String operationCode) {
            for (Operation o : Operation.values()) {
                if (o.getCode().equals(operationCode)) {
                    return o;
                }
            }
            throw new RuntimeException("Operation code is not supported: " + operationCode);
        }

Reply:
Yes, thanks, did this method - try to do it with stream))